### DISCUSSION TITLE
Datatable: Row edit with validation results in disabled inputfields (Primefaces 14.0.6)

### DISCUSSION THREAD

**Discussion by AndrewChurchill1970** [primefaces/community#3239] [PrimeFaces] [answered]
We have a validation check on editable mandatory row. If inputfield results in validation error (ui-state-error) the inputfield gets automatically disabled and can't be edited anymore. The inputfield get's automatically the attributes **disabled="disabled"** and **data-disabled-by-editor="true"**. Why and how to avoid that? it should be able to enter valid input after validation error. Now all columns are disabled.

**Comment by melloware** [comment]
It is a bug can you open a ticket on GitHub Issues please: https://github.com/primefaces/primefaces/issues

**Reply by AndrewChurchill1970** [reply, marked as answer]
https://github.com/primefaces/primefaces/issues/13091

thx4hlp!

**Reply by melloware** [reply]
Thanks PR submitted: https://github.com/primefaces/primefaces/pull/13093
